portfolio/src/main/webapp/script.js

```diff
@@ -48,12 +56,52 @@ function getMessages() {
   });
 }
 
+
+
+//check whether user is logged in using async
+async function checkWhetherUserIsLoggedIn() {
+  //set visibility of form and logurl to none
+  document.getElementById('comment-form').style.display="none";
+  document.getElementById('logInOut').style.display="none";
+  
+  //get login status
+  const response = await fetch('/login');
+  const quote = await response.text();
```

Comment:
This is no longer a quote, right? Maybe change the name to something like `loginInfo`?

Reply:
reusing code haha! that's true